code/parse/sexp.cpp

```diff
@@ -17145,20 +17142,6 @@ void sexp_set_countermeasures(int node)
 	Current_sexp_network_packet.end_callback();
 }
 
-void multi_sexp_set_countermeasures()
-{	
-	int num_cmeasures = 0;
-	ship *shipp; 
-
-	Current_sexp_network_packet.get_ship(shipp);
-	if (shipp == NULL) {
-		return;
-	}
-	if (Current_sexp_network_packet.get_int(num_cmeasures)) {
-		shipp->cmeasure_count = num_cmeasures;
-	}
-}
-
```

Comment:
This function should not be deleted.  Instead, `multi_sexp_set_countermeasures` should be added amidst the rest of the multi sexp calls.

Reply:
can you give me a hint what i need to do to add it to the appropriate list?

Reply:
I thought it wouldn't be hard to figure out, but sure.  Look in `multi_sexp_eval` where all the other multi_sexp_* functions are called.  Add the appropriate `case` statements and function calls to the big `switch` block.